docs/api-reference/classes/Config.mdx

```diff
@@ -0,0 +1,307 @@
+[testplane](../README.mdx) / Config
+
+# Class: Config
+
+## Extends
+
+- [`CommonConfig`](../interfaces/CommonConfig.mdx)
+
+## Constructors
+
+### new Config()
+
+```ts
+new Config(config?): Config
+```
+
+#### Parameters
+
+| Parameter | Type |
+| ------ | ------ |
+| `config`? | `string` \| [`ConfigInput`](../type-aliases/ConfigInput.mdx) |
+
+#### Returns
+
+[`Config`](Config.mdx)
+
+#### Defined in
+
+[src/config/index.ts:28](https://github.com/gemini-testing/testplane/blob/57d0b182c37e21d8bdf52acbb137a028841767af/src/config/index.ts#L28)
+
+## Properties
+
+| Property | Type | Inherited from | Defined in |
+| ------ | ------ | ------ | ------ |
+| `antialiasingTolerance` | `number` | [`CommonConfig`](../interfaces/CommonConfig.mdx).`antialiasingTolerance` | [src/config/types.ts:133](https://github.com/gemini-testing/testplane/blob/57d0b182c37e21d8bdf52acbb137a028841767af/src/config/types.ts#L133) |
+| `assertViewOpts` | [`AssertViewOptsConfig`](../interfaces/AssertViewOptsConfig.mdx) | [`CommonConfig`](../interfaces/CommonConfig.mdx).`assertViewOpts` | [src/config/types.ts:136](https://github.com/gemini-testing/testplane/blob/57d0b182c37e21d8bdf52acbb137a028841767af/src/config/types.ts#L136) |
+| `automationProtocol` | `"webdriver"` \| `"devtools"` | [`CommonConfig`](../interfaces/CommonConfig.mdx).`automationProtocol` | [src/config/types.ts:97](https://github.com/gemini-testing/testplane/blob/57d0b182c37e21d8bdf52acbb137a028841767af/src/config/types.ts#L97) |
+| `baseUrl` | `string` | [`CommonConfig`](../interfaces/CommonConfig.mdx).`baseUrl` | [src/config/types.ts:104](https://github.com/gemini-testing/testplane/blob/57d0b182c37e21d8bdf52acbb137a028841767af/src/config/types.ts#L104) |
+| `browsers` | `Record`\<`string`, `BrowserConfig`\> | - | [src/config/types.ts:213](https://github.com/gemini-testing/testplane/blob/57d0b182c37e21d8bdf52acbb137a028841767af/src/config/types.ts#L213) |
```

Comment:
BrowserConfig is referenced here, yet there is no definition. How is this resolved in general?